PhysicsTools/PatAlgos/python/slimming/packedPFCandidates_cfi.py

```diff
@@ -6,5 +6,8 @@
     originalVertices = cms.InputTag("offlinePrimaryVertices"),
     originalTracks = cms.InputTag("generalTracks"),
     vertexAssociator = cms.InputTag("primaryVertexAssociation","original"),
+    PuppiWeight = cms.InputTag("puppi","PuppiWeights"),
+    PuppiSrc = cms.InputTag("puppi"),
+    PuppiSrcMap = cms.InputTag("puppi"),
```

Comment:
PuppiWeights isnt needed since puppi only makes one float value map.
for the input tags for PuppiSrc and PuppiSrcMap, since the edmTokens where different, I made both. I guess I can change them to just one puppiSrc